doc/manuals/Substitute.am

```diff
@@ -17,6 +17,13 @@ SUBSTITUTE=sed "\
 	s%@SYSCONFDIR@%$(sysconfdir)%g;\
 	"
 
-## Example of how to substitute:
-## squid.8: $(srcdir)/squid.8.in Makefile
-##	$(SUBSTITUTE) < $(srcdir)/squid.8.in > $@
+if ENABLE_PANDOC_DOC
+## Due to use of $(SUBSTITUTE), built .8 files need to explicitly
+## depend on Makefile. That dependency should be declared at the
+## location where a .8 file is built. Due to the suffix rule below,
+## that explicit .8 file dependency list should not include an .md file.
+## See squid.8 in src/Makefile.am for an example to follow.
+.md.8:
+	$(SUBSTITUTE) < $(srcdir)/`basename $<` | \
```

Comment:
Does using bare `$<` work here? We do have other seemingly similar rules (including rules in included src/TestHeaders.am) that avoid `$(srcdir)/basename` complications...

```suggestion
	$(SUBSTITUTE) < $< | \
```